Molly/Molly/README.md

```diff
@@ -0,0 +1,273 @@
+### Homework 5
+
+_Due Mon. Aug.17_
+
+####Synopsis
+
+- **Problem 1:** A Cards Module _[20% of total time]_ **Goals:** Start working with IIFEs and closure to create a self-contained module for your cards.
+- **Problem 2:** All Hands Off Deque _[25%]_ **Goals:** Get more comfortable with closure by creating a secure version of the deque from last week.
+- **Problem 3:** Secrets At All Levels _[25%]_ **Goals:** Build on Monday's in-class password example to practice closures at the level of both instance and factory methods.
+- **Problem 4:** Coming soon!
+
+---
+
+
+---
+
+**1)  Card module**
```

Comment:
Everything here is working well. If you could do your TA's one slight favor and place your code answers in a separate javascript file it would be tremendously helpful in testing! All in all, good work on question one.